digit-health-services/health-delivery-service/src/main/java/org/digit/health/delivery/web/models/response/DeliveryResponse.java

```diff
@@ -0,0 +1,18 @@
+package org.digit.health.delivery.web.models.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.egov.common.contract.response.ResponseInfo;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class DeliveryResponse {
+
+    @JsonProperty("responseInfo")
+    private boolean status;
```

Comment:
@JsonProperty("responseInfo")
private ResponseInfo responseInfo;

Can we change

Reply:
updated.